components/core/src/streaming_archive/LogtypeSizeTracker.cpp

```diff
@@ -0,0 +1 @@
+#include "LogtypeSizeTracker.hpp"
```

Comment:
Let's delete this file (our practices of creating empty cpp files have changed).